src/core/demangle.d

```diff
@@ -208,11 +208,11 @@ pure @safe:
         {
             assert( contains( dst[0 .. len], val ) );
             debug(info) printf( "removing (%.*s)\n", cast(int) val.length, val.ptr );
-
             size_t v = &val[0] - &dst[0];
+            assert(len >=val.length && len <= dst.length);
```

Comment:
Missing space in `>=val`.

Reply:
done

Reply:
Not sure if this assert is necessary, it is also implied by `contains` above.

Reply:
it's a bit more straightforward with the assert ; shouldn't affect performance even with asserts enabled given the rest of the code in this function; if you insist I'll remove this assert